### ISSUE TITLE
Use query stream and get some broken symbols

### ISSUE THREAD

**Issue by AlekseyVerba** [ClickHouse/clickhouse-js#171]
### Describe the bug
I use query stream and get broken symbols.  In database data is normal, without any defects. When i get data from stream, i get something strange

i get these rows:
"[\"ᴺᵁᴸᴸ\", \"45049\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"Сайт\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺ���ᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"256\", \"0\", \"spb.evraz.market\", \"Каталог товаров\", \"Пользователь\", \"\\/metalloprokat\\/\", \"0.256\", \"Время ответа сайта\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\"]"

"[\"A89648DF371528B911EB7CC47A32031D\", \"45027\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"Проведе��ие заказа\", \"A8D548DF371528B911ED9BDB5E11ABCD\", \"ᴺᵁᴸᴸ\", \"ИН00-047642\", \"ИН00-040371 от 21.02.2022 16:43:08\", \"ᴺᵁᴸᴸ\", \"A8B948DF371528B911EC7D1294FB667E\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"1925\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"3.154\"]"

i tried to use another library and i didn't get these mistakes

### Expected behaviour
"[\"ᴺᵁᴸᴸ\", \"45049\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"Сайт\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"256\", \"0\", \"spb.evraz.market\", \"Каталог товаров\", \"Пользователь\", \"\\/metalloprokat\\/\", \"0.256\", \"Время ответа сайта\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\"]"

"[\"A89648DF371528B911EB7CC47A32031D\", \"45027\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"Проведение заказа\", \"A8D548DF371528B911ED9BDB5E11ABCD\", \"ᴺᵁᴸᴸ\", \"ИН00-047642\", \"ИН00-040371 от 21.02.2022 16:43:08\", \"ᴺᵁᴸᴸ\", \"A8B948DF371528B911EC7D1294FB667E\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"1925\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"ᴺᵁᴸᴸ\", \"3.154\"]"

### Code example

`
        const ch_query = await ch.query({
            query: queryText,
            format: 'JSONCompactStringsEachRow',
        });

        const ch_stream = await ch_query.stream();

        const transformStream = new Transform({
            transform(data, encoding, callback) {
                data.forEach((element) => {
                    this.push(element.json());
                });
                callback();
            },
            objectMode: true,
        });
        
        transformStream.on('data', data => {
            appendFileSync('text.txt', JSON.stringify(data) + '\n');
        })

`

#### Environment
* Client version: 0.0.16
* Language version: v16.18.1
* OS: Ubuntu 22.04

#### ClickHouse server
* ClickHouse Server version: 23

**Comment by slvrtrn** [comment]
Can you please provide table DDL + sample dataset?

**Comment by slvrtrn** [comment]
Additionally, at a glance, Transform might be unnecessary here.

```ts
const ch_query = await ch.query({
  query: queryText,
  format: 'JSONCompactStringsEachRow',
})

ch_query.stream().on('data', (rows: Row[]) => {
  let content = ''
  rows.forEach((row) => {
    content += row.text + os.EOL
  })
  fs.appendFileSync('text.txt', content)
})
```

**Comment by slvrtrn** [comment]
@AlekseyVerba, can you please contact me in the community Slack? I think we will need the entire dataset to debug this then.

**Comment by slvrtrn** [comment]
@AlekseyVerba, join the community Slack ([link](https://clickhouse.com/slack)) and find me (Serge Klochkov) there.

**Comment by slvrtrn** [comment]
@AlekseyVerba, fixed in 0.2.4.
